src/psyclone/psyGen.py

```diff
@@ -1178,11 +1208,18 @@ def gen_code(self, parent):
 class Directive(Node):
 
     def view(self, indent=0):
-        print self.indent(indent) + "Directive"
+        ''' Print a text representation of this node to stdout '''
```

Comment:
Added in a comment string. Whoop. You haven't done so in all cases where a view method is changed but it is not a requisite for this issue so any improvement is a bonus.